app/static/lib/conp-react/src/DatasetElement/index.js

```diff
@@ -48,250 +47,259 @@ const DatasetElement = (props) => {
-                      </li> : null}
-                  {element.dateAdded ?
-                      <li className="card-list-item">
-                        <strong>Date Added: </strong>{element.dateAdded}
-                      </li> : null}
-                  {element.dateUpdated ?
-                      <li className="card-list-item">
-                        <strong>Date Updated: </strong>{element.dateUpdated}
-                      </li> : null}
-                </ul>
-                <ul className="d-flex align-items-start">
-                  {element.types ?
-                      <li className="card-list-item">
-                        <strong>Data Types: </strong>{element.types}
-                      </li> : null}
-                  {element.modalities ?
-                      <li className="card-list-item">
-                        <strong>Modalities: </strong>{element.modalities.join(', ')}
-                      </li> : null}
-                  {element.licenses ?
-                      <li className="card-list-item">
-                        <strong>License: </strong>{element.licenses}
-                      </li> : null}
-                </ul>
-                <ul className="d-flex align-items-start">
-                  {element.files ?
-                      <li className="card-list-item">
-                        <strong>Files: </strong>{element.files}
-                      </li> : null}
-                  {element.size ?
-                      <li className="card-list-item">
-                        <strong>Size: </strong>{element.size}
-                      </li> : null}
-                  {element.subjects ?
-                      <li className="card-list-item">
-                        <strong>Subjects: </strong>{element.subjects}
-                      </li> : null}
-                  {element.formats ?
-                      <li className="card-list-item">
-                        <strong>Formats: </strong>{element.formats.join(', ')}
-                      </li> : null}
-                </ul>
-              </div>
-              <div>
-                <ArkIdElement id={element.ark_id}/>
-              </div>
+        </div>
+        <div className="col col-lg-3 d-flex flex-column justify-content-center align-items-center p-2">
+          <div className="row align-items-center justify-content-center w-100">
+            <div className="col-10 p-0">
+              {element.showDownloadButton ?
+                <button
+                  type="button"
+                  className="btn btn-success m-1"
+                  onClick={() => openDownloadModal()}
+                >
+                  Direct Download ({element.size})
+                </button> :
+                <div className="btn btn-outline-secondary m-1 disabled">
+                  Direct Download (Not Available)
+                </div>
+              }
+            </div>
+            <div className="col-2 p-2">
+              <p className="card-text pl-1">
```

Comment:
Maybe make this a `div`? React complains for me in the dev console about having a `ReactToolTip` as a descendant of a `p`.